Doppler.Currency/Services/TrmHandler.cs

```diff
@@ -45,28 +48,27 @@ public override async Task<EntityOperationResult<CurrencyDto>> Handle(DateTime d
             return await GetDataFromHtmlAsync(jsonContent, date);
         }
 
-        private async Task<EntityOperationResult<CurrencyDto>> GetDataFromHtmlAsync(string jsonContent, DateTime date)
+        private Task<EntityOperationResult<CurrencyDto>> GetDataFromHtmlAsync(string jsonContent, DateTime date)
         {
             var result = new EntityOperationResult<CurrencyDto>();
-            dynamic data = JsonConvert.DeserializeObject(jsonContent);
+            IList<TrmResponse> data = JsonConvert.DeserializeObject<IList<TrmResponse>>(jsonContent);
```

Comment:
you can use `var`

Reply:
Yes, do you prefer that I use "var"?

Reply:
Yes, but as you prefer

Reply:
I will update the line to be consist with the above line.